src/AppNexusClient.php

```diff
@@ -3,11 +3,11 @@
 
 class AppNexusClient
 {
-    private $http;
-    private $host;
-    private $username;
-    private $password;
-    private $tokenStorage;
+    protected $http;
```

Comment:
What do you need inheritance for? I'd prefer to use composition over inheritance to stick to Open-Close principle.